src/cpp/rtps/builtin/discovery/participant/PDPSimple.cpp

```diff
@@ -256,14 +256,19 @@ bool PDPSimple::initPDP(RTPSParticipantImpl* part)
         return false;
     }
 
-    if(!mp_RTPSParticipant->enableReader(mp_SPDPReader))
-        return false;
-
-    mp_resendParticipantTimer = new ResendParticipantProxyDataPeriod(this, m_discovery);
-
     return true;
 }
 
+bool PDPSimple::enable()
+{
+    bool result = mp_RTPSParticipant->enableReader(mp_SPDPReader);
+    if (result)
+    {
+        mp_resendParticipantTimer = new ResendParticipantProxyDataPeriod(this, m_discovery);
+    }
+    return result;
```

Comment:
Master is different now, we are not resending the participant proxy data, can you check when and why this was removed? I couldn't find it...

Reply:
This is needed in the backport due to the lack of the event system. In the new code was moved to `PDP::initPDP`.

Reply:
:+1: